docs/api-reference/carto/basemap.md

```diff
@@ -22,7 +22,7 @@ npm install @deck.gl/core @deck.gl/layers @deck.gl/carto
 
 ```js
 import {DeckGL} from '@deck.gl/react';
-import {StaticMap} from 'react-map-gl';
+import {StaticMap} from 'react-map-gl/maplibre';
```

Comment:
`StaticMap` was a component from older versions. It has be replaced by just `Map`.